generate-stackbrew-library.sh

```diff
@@ -17,14 +18,14 @@ echo '# maintainer: InfoSiftr <github@infosiftr.com> (@infosiftr)'
 for version in "${versions[@]}"; do
 	commit="$(git log -1 --format='format:%H' -- "$version")"
 	fullVersion="$(grep -m1 'ENV REDIS_VERSION ' "$version/Dockerfile" | cut -d' ' -f3)"
-	
+
 	versionAliases=()
 	while [ "$fullVersion" != "$version" -a "${fullVersion%[.-]*}" != "$fullVersion" ]; do
 		versionAliases+=( $fullVersion )
 		fullVersion="${fullVersion%[.-]*}"
 	done
 	versionAliases+=( $version ${aliases[$version]} )
-	
+
```

Comment:
@Starefossen, We just like consistency in all of the `docker-library/` repos, so just a matter of preference ala [bikeshedding](http://bikeshed.io).  If there are inconsistencies, I blame @tianon for his laxed merging standards (it totally was never me :stuck_out_tongue_winking_eye:).